src/DynamoCoreWpf/UI/GuidedTour/HostControlInfo.cs

```diff
@@ -94,5 +96,16 @@ public double HorizontalPopupOffSet
                 horizontalPopupOffSet = value;
             }
         }
+
+        /// <summary>
+        /// Defines a delta width fot the highlighted box
+        /// </summary>
+        [JsonProperty("WidthBoxDelta")]
+        public double WidthBoxDelta { get => widthBoxDelta; set => widthBoxDelta = value; }
```

Comment:
Can you explain more why we needed the delta width and height?

Reply:
Hi @QilongTang, sorry I wasn't clear. 
Aabishkar commented that the box isn't at the right height.
Since the size of the box is defined by the size of the hosted element, the values are fixed.
A way I thought to solve this is to create a delta value that increases or decreases the width or height of the box whenever necessary.

Reply:
@filipeotero Sure, would you put these as comments there in addition to what you had?